src/trumbowyg.js

```diff
@@ -153,10 +153,16 @@ jQuery.trumbowyg = {
         t.hideButtonTexts = $.trumbowyg.hideButtonTexts != null ? $.trumbowyg.hideButtonTexts : options.hideButtonTexts;
 
         // SVG path
-        var svgPathOption = $.trumbowyg.svgPath != null ? $.trumbowyg.svgPath : options.svgPath;
+        var svgPathOption = $trumbowyg.svgPath != null ? $trumbowyg.svgPath : options.svgPath;
         t.hasSvg = svgPathOption !== false;
-        t.svgPath = !!t.doc.querySelector('base') ? window.location.href.split('#')[0] : '';
-        if ($('#' + trumbowygIconsId, t.doc).length === 0 && svgPathOption !== false) {
+        var sideLoad = options.svgSideLoad === false ? false : true;
```

Comment:
A Cleaner way is: 

    var sideLoad = options.svgSideLoad !== false;

Reply:
I was just trying to assure that it was in fact a boolean and not relying on the truthiness downstream. You're right about the simplification, thanks.